crates/ironrdp-pdu/src/rdp/capability_sets/general.rs

```diff
@@ -25,9 +24,45 @@ pub enum MajorPlatformType {
     OsX = 6,
     Android = 7,
     ChromeOs = 8,
+    Other(u16),
 }
 
-#[derive(Copy, Clone, Debug, PartialEq, Eq, FromPrimitive, ToPrimitive)]
+impl From<u16> for MajorPlatformType {
+    fn from(value: u16) -> Self {
+        match value {
+            0 => Self::Unspecified,
+            1 => Self::Windows,
+            2 => Self::Os2,
+            3 => Self::Macintosh,
+            4 => Self::Unix,
+            5 => Self::IOs,
+            6 => Self::OsX,
+            7 => Self::Android,
+            8 => Self::ChromeOs,
+            _ => Self::Other(value),
+        }
+    }
+}
+
+impl Into<u16> for MajorPlatformType {
```

Comment:
`From` should be implemented over `Into`
```suggestion
impl From<MajorPlatformType> for u16 {
```

[Here is why](https://doc.rust-lang.org/std/convert/trait.Into.html#):
```
/// One should always prefer implementing `From` over [`Into`]
/// because implementing `From` automatically provides one with an implementation of [`Into`]
/// thanks to the blanket implementation in the standard library.
///
/// Only implement [`Into`] when targeting a version prior to Rust 1.41 and converting to a type
/// outside the current crate.
/// `From` was not able to do these types of conversions in earlier versions because of Rust's
/// orphaning rules.
```

I believe CI is failing because of the clippy lint for this.

Reply:
done